core/block_render_svg_vertical.js

```diff
@@ -507,6 +508,26 @@ Blockly.BlockSvg.prototype.renderCompute_ = function(iconWidth) {
     } else {
       input.renderWidth = 0;
     }
+
+    // If the input is a statement input, determine if a notch
+    // should be drawn at the inner bottom of the C.
+    if (input.connection && input.connection.type === Blockly.NEXT_STATEMENT) {
+      // Walk down linkedBlock's nextConnection to determine if
```

Comment:
This should be a helper function in block.js (maybe getLastBlockInStack()), not implemented inline here.

Looks like the android one is called getLastBlockInSequence()
https://github.com/google/blockly-android/blob/6c4d4376c1390d73d67a13375f72a68dbb341444/blocklylib-core/src/main/java/com/google/blockly/model/Block.java#L568